packages/fluentui/react-northstar/src/themes/teams/components/Form/formStyles.ts

```diff
@@ -6,12 +6,13 @@ const formStyles: ComponentSlotStylesPrepared<FormStylesProps, any> = {
   root: ({ props, variables }): ICSSInJSStyle => ({
     height: '100%',
     width: '100%',
-    display: ['grid', '-ms-grid'],
-    gridTemplateColumns: '1fr',
-    msGridColumns: '1fr',
+    display: 'flex',
+    flexDirection: 'column',
     justifyContent: 'space-evenly',
-    gridGap: pxToRem(20),
     justifyItems: 'start',
+    '> *:not(:last-child)': {
```

Comment:
nit; if you're doing gap, and you render a single item, that should not have marginBottom, correct?

I usually do `:not(:first-child) { marginTop: ... }` for gaps.

Reply:
Hmm good point, I wonder if we wouldn't always want that form have the `marginBottom` I think if we remove it we might get some regression, since what I was trying to do there was just achieve same result we have before but working on IE also.. I will apply your suggestion and see if screener catches something :)